Assets/Scripts/Map/PatrollingMap.cs

```diff
@@ -15,7 +15,7 @@ public sealed class PatrollingMap : ICloneable<PatrollingMap>
     {
         public readonly IReadOnlyList<Vertex> Vertices;
```

Comment:
I would really like if this one was removed

Reply:
Sounds fine to me, but its unrelated to this PR, make an issue and we can maybe have a look at it in another PR, theres a lot of places that property is accessed, so it would take some work to remove it.